app/views/visitor_mailer/request_acknowledged.html.erb

```diff
@@ -0,0 +1,70 @@
+<p>
+  <%= t('.salutation', visitor_full_name: @visit.visitor_first_name) %>,
+</p>
+<h1>
+  <%= t('.status') %>
+</h1>
+<p>
+  <%= t('.where_to_check_status') %>
+  <%= link_to('TODO show booking', '#') %>
+</p>
+<p>
+  <%= t('.when_to_expect_response', when: 'TODO') %>
+</p>
+<p>
+  <%= t('.when_to_check_spam', when: 'TODO') %>
+</p>
+<h1>
+  <%= t('.add_to_address_book') %>
+</h1>
+<p>
+  <%= t('.add_address', email: 'no-reply@email.prisonvisits.service.gov.uk') %>
+</p>
+<h1>
+  <%= t('.details') %>
+</h1>
+<p>
+  <%= t('.request',
+        prison: @visit.prison_name,
+        number_of_visitors: pluralize(@visit.total_number_of_visitors,
+                                      'person',
+                                      'people')
```

Comment:
These English words don't belong in the template. If you use `count` as the variable, you can use `one`/`other` in the locale file to deal with plurals.